### PR TITLE
pgbadger 10.1

### PR DESCRIPTION
**Pull request by SteveNewson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/pgbadger.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Pgbadger < Formula
 
   def install
     system "perl", "Makefile.PL", "DESTDIR=#{buildpath}"
+    inreplace "Makefile", "sed -i '", "sed -i '' '"
```
Comment (fxcoudert) [inline Formula/pgbadger.rb]: This issue needs to be reported and fixed by upstream.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/pgbadger.rb (hunk 1 above)
Comment (SteveNewson) [reply Formula/pgbadger.rb]: Will do

Comment (SteveNewson) [reply Formula/pgbadger.rb]: Change merged upstream. Now awaiting a release:

https://github.com/darold/pgbadger/pull/438

**Comment by SteveNewson** [comment]
@fxcoudert Upstream changes have been made and incorporated into 10.1. This PR is now to pull in 10.1 of pgbadger without a manual `inreplace` patch.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks a lot @SteveNewson for the pull request and follow-up!

**Pull request closed without merging**